module-audio/Audio/Profiles/ProfileConfigUtils.cpp

```diff
@@ -87,6 +89,29 @@ namespace audio
         config.inputPath     = utils::toEnum<codec::InputPath>(configJson[strings::inputPath].int_value());
         config.outputPath    = utils::toEnum<codec::OutputPath>(configJson[strings::outputPath].int_value());
 
+        std::uint8_t playbackPathGain  = configJson[strings::playbackPathGain].int_value();
+        std::uint8_t playbackPathAtten = configJson[strings::playbackPathAtten].int_value();
+
+        constexpr std::uint8_t playbackPathGainMax = 3;
```

Comment:
would be nice to have such absolute maximum in the header, rather than in cpp but I see that it's a standard in this file - so it's just a notice :) please close after reading ;)

Reply:
I'll do it that way next time, thanks for the info!